test/conftest.py

```diff
@@ -151,10 +152,11 @@ def test_reports_engine(monkeypatch):
 
 
 @pytest.fixture
-def test_reports_session(test_reports_engine):
+def test_reports_session(test_reports_engine, monkeypatch):
     """
     Returns a sqlalchemy session for the test database.
     """
+    monkeypatch.setattr(publication, "RIALTO_REPORTS_DB_NAME", "rialto_reports_test")
```

Comment:
Moving this monkeypatching here means we don't need to remember to do it in tests involving the reports database.